src/device/wg_controller.rs

```diff
@@ -1483,25 +1571,36 @@ mod tests {
         let pub_key_1 = SecretKey::gen().public();
         let pub_key_2 = SecretKey::gen().public();
 
-        let mut requested_state = create_requested_state(vec![(pub_key_1, vec![], false, false)]);
+        let mut requested_state =
+            create_requested_state(vec![(pub_key_1, vec![], false, false, false, false)]);
 
         requested_state.exit_node = Some(ExitNode {
             public_key: pub_key_2,
             ..Default::default()
         });
 
-        expect_add_to_peer_whitelist(&mut firewall, pub_key_starcast_vpeer);
+        expect_add_to_vpeer_whitelist(&mut firewall, pub_key_starcast_vpeer);
+
+        firewall.expect_set_ip_addresses().once().return_const(());
 
         expect_get_peer_whitelist(&mut firewall, vec![pub_key_1]);
         expect_get_port_whitelist(&mut firewall, vec![]);
 
+        println!("Peer1 {:?} peer2 {:?}", pub_key_1, pub_key_2);
+
         expect_remove_from_peer_whitelist(&mut firewall, pub_key_1);
 
-        expect_add_to_peer_whitelist(&mut firewall, pub_key_2);
+        // expect_add_to_peer_whitelist(&mut firewall, pub_key_2);
```

Comment:
C'mon, commenting out code to remove it is not what you do in production code.

Reply:
Removed.